ee/app/livechat-enterprise/server/startup.js

```diff
@@ -32,5 +34,17 @@ Meteor.startup(async function() {
 			businessHourManager.startManager();
 		}
 	});
+	settings.onload('Livechat_max_queue_wait_time_action', function(_, value) {
+		updateQueueInactivityTimeout();
+		if (!value || value === 'Nothing') {
+			return queueInactivityMonitor.stop();
+		}
+		return queueInactivityMonitor.start();
+	});
+
+	settings.onload('Livechat_max_queue_wait_time', function() {
```

Comment:
What if `Livechat_max_queue_wait_time_action` === 0? Should `updateQueueInactivityTimeout` be called?